config/locales/views/organizations/en.yml

```diff
@@ -72,8 +72,9 @@ en:
       group_assignment: 'Group assignment'
       group_assignment_desc: 'Participants form teams that work together on a shared repository.'
       create_group_assignment: 'Create a group assignment'
-      name_your_classroom: 'Name your classroom'
-      give_descriptive_name: 'Please give your classroom a more descriptive name'
+      name_your_classroom: 'Set up your classroom'
+      give_descriptive_name: 'Classroom name'
+      classroom_name_note: 'Try using your course name and section so students can identify your classroom.'
```

Comment:
"Try" kind of sounds like we don't allow anything else, would something like "consider" sound like less of an imperative?

Reply:
"Using your course name and section can help students identify your classroom." is shorter if that helps? Sorry, I just assume that I don't have a choice when someone tells me to try something 😄